scene/gui/label.cpp

```diff
@@ -673,6 +673,19 @@ Size2 Label::get_minimum_size() const {
 	}
 }
 
+#ifndef DISABLE_DEPRECATED
+bool Label::_set(const StringName &p_name, const Variant &p_value) {
+	if (p_name == SNAME("valign")) {
+		vertical_alignment = static_cast<VerticalAlignment>(p_value.operator int());
+		return true;
+	} else if (p_name == SNAME("align")) {
+		horizontal_alignment = static_cast<HorizontalAlignment>(p_value.operator int());
```

Comment:
```suggestion
		set_vertical_alignment(static_cast<VerticalAlignment>(p_value.operator int()));
		return true;
	} else if (p_name == SNAME("align")) {
		set_horizontal_alignment(static_cast<HorizontalAlignment>(p_value.operator int()));
```
My bad realized this would be safer

Reply:
If changing this, usually we just use C-style casts. I'm not sure `static_cast` adds much value here.
```suggestion
		set_vertical_alignment((VerticalAlignment)p_value.operator int());
		return true;
	} else if (p_name == SNAME("align")) {
		set_horizontal_alignment((HorizontalAlignment)p_value.operator int());
```